etc-conf/dbus/system.d/com.redhat.SubscriptionManager.conf

```diff
@@ -8,12 +8,35 @@
   <policy user="root">
     <allow own="com.redhat.SubscriptionManager"/>
     <allow own="com.redhat.SubscriptionManager.PluginEvent"/>
+
+    <!-- Only root can trigger methods in these services -->
+    <allow send_destination="com.redhat.SubscriptionManager"/>
+    <allow send_destination="com.redhat.SubscriptionManager.PluginEvent"/>
+
+    <!-- Basic D-Bus API stuff -->
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.Introspectable"/>
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.Properties"/>
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.ObjectManager"/>
   </policy>
 
   <policy context="default">
-    <allow send_destination="com.redhat.SubscriptionManager"/>
-    <allow send_destination="com.redhat.SubscriptionManager.EntitlementStatus"/>
+    <!-- Common user can only check status of entitlement -->
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_path="/EntitlementStatus"
+        send_member="check_status"/>
+
+    <!-- Basic D-Bus API stuff -->
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.Introspectable"/>
+    <allow send_destination="com.redhat.SubscriptionManager"
+        send_interface="org.freedesktop.DBus.Properties"/>
```

Comment:
OK, I just copy pasted this `org.freedesktop.DBus` part from other conf files in `etc-conf/dbus/system.d/`. Should I concern about other conf files too?